sklearn/utils/murmurhash.pyx

```diff
@@ -89,12 +89,12 @@ def murmurhash3_32(key, seed=0, positive=False):
     Parameters
     ----------
     key : int32, bytes, unicode or ndarray with dtype int32
```

Comment:
Although this is a quite complex, maybe the following is better:

```suggestion
    key : int32, bytes, unicode or ndarray, dtype=int32
```

or

```suggestion
    key : int32, bytes, unicode or ndarray, dtype=np.int32
```

WDYT?